addon/tailwind/config/modules.css

```diff
@@ -139,6 +139,7 @@ pre {
 .overflow-momentum {
   -webkit-overflow-scrolling: touch;
 }
-.outline-none {
+.outline-none,
+.outline-none:focus {
```

Comment:
Had to also override the default for the `:focus` state, otherwise it would be outlines while clicked on and then lose focus. Probably not the visual that is desired.